src/_reducers/TickReducer.js

```diff
@@ -17,7 +17,7 @@ export const mergeTicks = (existingTicks, newTicks) => {
 
     const lastNewTicksEpoch = getLast(newTicks).epoch;
     const oldestExistingTickEpoch = existingTicks[0].epoch;
-    const lastExistingTickEpoch = getLastTick(existingTicks).epoch;
+    const lastExistingTickEpoch = getLast(existingTicks).epoch;
```

Comment:
it at least serve as a shorthand of

superUglyAndLongVariableName[superUglyAndLongVariableName.length - 1]